index.bs

```diff
@@ -3980,16 +3980,20 @@ When registering a new credential, represented by an {{AuthenticatorAttestationR
 
 1. Verify that the values of the [=client extension outputs=] in |clientExtensionResults| and the [=authenticator extension
     outputs=] in the <code>[=authdataextensions|extensions=]</code> in |authData| are as expected, considering the [=client
-    extension input=] values that were given in <code>|options|.{{PublicKeyCredentialCreationOptions/extensions}}</code>.
-    In particular, any [=extension identifier=] values in the |clientExtensionResults| and
-    the <code>[=authdataextensions|extensions=]</code> in |authData|
-    MUST also be present as [=extension identifier=] values in
-    <code>|options|.{{PublicKeyCredentialCreationOptions/extensions}}</code>,
-    i.e., no extensions are present that were not requested.
+    extension input=] values that were given in <code>|options|.{{PublicKeyCredentialCreationOptions/extensions}}</code>
+    and any specific policy of the [=[RP]=] regarding unsolicited extensions, i.e. those that were not specified as part of 
+    <code>|options|.{{PublicKeyCredentialCreationOptions/extensions}}</code>.
     In the general case, the meaning of "are as expected" is specific to the [=[RP]=] and which extensions are in use.
 
-    Note: Since all extensions are OPTIONAL for both the [=client=] and the [=authenticator=], the [=[RP]=] MUST be prepared to
-    handle cases where none or not all of the requested extensions were acted upon.
+    Note:  [=Client platforms=] MAY enact local policy that sets additional [=authenticator extensions=] or
+    [=client extensions=] and thus cause values to appear in the [=authenticator extension outputs=] or
+    [=client extension outputs=] that were not originally specified as part of 
+    <code>|options|.{{PublicKeyCredentialCreationOptions/extensions}}</code>. [=[RPS]=] should be aware of this and 
+    be able to handle such situations, whether it be to ignore the unsolicited extensions or reject the attestation. The
```

Comment:
```suggestion
    prepared to handle such situations, whether it be to ignore the unsolicited extensions or reject the attestation. The
```

Reply:
Resolved in https://github.com/w3c/webauthn/pull/1289/commits/dc0378b9dd1aa142296ee25021e3b511e5b078c0